backend/infrahub/core/diff/payload_builder.py

```diff
@@ -31,7 +37,9 @@ async def get_display_labels(nodes: dict[str, dict[str, list[str]]], db: Infrahu
         if branch_name not in response:
             response[branch_name] = {}
         for kind, ids in items.items():
-            labels = await get_display_labels_per_kind(kind=kind, ids=ids, db=db, branch_name=branch_name)
+            labels = await get_display_labels_per_kind(
+                kind=kind, ids=ids, db=db, branch_name=branch_name, skip_missing_schema=True
+            )
             response[branch_name].update(labels)
 
     return response
```

Comment:
Changes in this file correspond to a cherry pick of https://github.com/opsmill/infrahub/pull/5128